test/integration/infra.go

```diff
@@ -156,19 +156,21 @@ func (infra *infra) setup() error {
 	//
 	// See github.com/kubernetes/kubernetes/issues/49048 for k8s
 	// tracking issue.
-	if yaml, err := fill("initializer.yaml.tmpl", infra); err != nil {
-		return err
-	} else if err = infra.kubeDelete(yaml, infra.IstioNamespace); err != nil {
-		glog.Infof("Sidecar initializer could not be deleted: %v", err)
-	}
+	if infra.UseInitializer {
```

Comment:
Why the change? InitializerConfiguration is clustered scope so each namespace needs to managed by an initializer even if the injection policy OFF. These were intentionally deployed regardless of InitializerConfiguration to avoid completely breaking shared test clusters.

Reply:
reverted